spec/index.html

```diff
@@ -2674,8 +2674,12 @@ <h2>Serialization</h2>
 <section id="privacy-considerations" class="informative">
   <h2>Privacy Considerations</h2>
 
-  <section>
-    <h3>Selective Disclosure Schemes</h3>
+  <p>Privacy considerations here are primarily worth discussing when the canonicalization scheme is used for
+    privacy-respecting signed RDF dataset and are likely acceptable for other use cases. One of the former examples is a
+    verifiable credential with selective disclosure.</p>
```

Comment:
Here's an idea about how we might frame the introduction to this section.

```suggestion
  <p>In general, RDF datasets are used for representing and conveying arbitrary information,
    which may include personally identifiable information which may be used for correlating
    other with data and disclose arbitrary information.
    After canonicalizing a dataset, even if some information is removed,
    it may be possible for a third party to infer omitted data.
    Applications should consider the potential for exposing personally identifiable information when constructing and conveying datasets.</p>
```